src/modules/feedback/feedback.service.ts

```diff
@@ -242,10 +242,16 @@ export class FeedbackService {
 
   async feedbackFiltered(
     query: FeedbackQueryDto,
-    user: UserSchemaType,
+    userId: string,
+    workspaceId: string | undefined,
   ): Promise<FeedbackFilteredResponseDto> {
     const { sentiment, limit, page } = query;
-    const whereConditions = [eq(schema.usersFeedbacksSchema.userId, user.id)];
+    const whereConditions = [
+      eq(schema.usersFeedbacksSchema.userId, userId),
+      workspaceId
+        ? eq(schema.usersFeedbacksSchema.workspaceId, workspaceId)
+        : undefined,
+    ];
```

Comment:
```suggestion
    const whereConditions = workspaceId ?  [eq(schema.usersFeedbacksSchema.userId, userId),eq(schema.usersFeedbacksSchema.workspaceId, workspaceId)] : [eq(schema.usersFeedbacksSchema.userId];
```

don't forget to format